roles/neutron-common/templates/etc/neutron/plugins/ml2/ml2_plugin.ini

```diff
@@ -45,6 +45,7 @@ vxlan_group = {{ neutron.vxlan.group_prefix }}
 [vxlan]
 enable_vxlan = {{ "vxlan" in neutron.tunnel_types }}
 l2_population = {{ neutron.l2_population }}
+arp_responder = True
```

Comment:
Would there be any value in making this a variable that defaults to True, so that we could experiment with it being off in some environments?